userspace/libsinsp/filter/ast.h

```diff
@@ -0,0 +1,292 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#pragma once
+
+#include <vector>
+#include <string>
+#include <algorithm>
+#include "../sinsp_public.h"
+
+namespace libsinsp {
+namespace filter {
+
+using namespace std;
```

Comment:
I think it's a bad practice to have "using namespace"s in header files, as anything that includes this header file, or any file that includes it recursively, will pick up the "using namespace" when they might not expect to.

Reply:
Agree on this, will move it!